soccer/gameplay/skills/dribble.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, pos=None):
         self._threshold = 0.1  #default value matches the required accuracy for a placement command
         self._pos = pos
 
-        self._dribble_speed = 100
+        self._dribble_speed = 70
```

Comment:
@jgkamat When I first wrote this I was just guessing that 100 would be good. For now I will set it to maxDribbleSpeed although it may need its own dribble value to keep the ball from having too much backspin when it tries to let go of it. I will address that with the rest of #676 when I work on that.